index.html

```diff
@@ -57,7 +57,7 @@
           <a href="https://twitter.com/ElijahHorton" target="_blank" rel="nofollow">@ElijahHorton</a></span>
         </div>
         <div class="heroku-version">
-          <a class="link-to-heroku" href="http://phpepl.herokuapp.com">Click for a PHP 5.5 version</a>
+          <a class="link-to-heroku" href="http://phpepl.herokuapp.com">Click for a PHP 5.6 version</a>
```

Comment:
Thanks for this! What do you think about just making it `5.x` to avoid having to maintain this number?